tracker/enqueue_test.go

```diff
@@ -70,7 +71,7 @@ func TestHappyPathsExact(t *testing.T) {
 	// Not tracked yet
 	{
 		trk.OnChanged(thing1)
-		if got, want := calls, 0; got != want {
+		if got, want := int(calls), 0; got != want {
```

Comment:
You can have want=int32(0) and increment it along the way, this avoiding the  ugly casts in every check

Reply:
I am trying a different approach now.